gitpod-io/website
# [Single Cluster Ref Arch] Aligns node pool max size across GCP and AWS

## Description
This PR aligns the max size of the node pools for the single cluster reference architecture across AWS and GCP towards: 
- max of 4 nodes for the service node pool
- Max of 50 nodes for the workspaces node pool

## How to test
- Make sure the actual change works: go to netlify > docs> self-hosted>installation-guides>reference-architectures>single cluster reference architecture
- Answer the following questions:
  - Is 4 nodes enough for the services node pool
  - Is 50 nodes enough or too much or just right for the workspace node pool?
  - is this the right approach or should these be different across GCP and AWS?

## Release Notes

```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2554"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
